tesseract_core/sdk/engine.py

```diff
@@ -704,7 +688,77 @@ def _parse_option(option: str):
             source = str(Path(source).resolve())
         return source, {"bind": target, "mode": mode}
 
-    return dict(_parse_option(opt) for opt in options)
+    volumes = {}
+    for opt in options:
+        source, opt_dict = _parse_option(opt)
```

Comment:
Should we have a better name for `_parse_option`? Based on reading the code here I can't really figure out what this function does or what it returns.